project/plugins.sbt

```diff
@@ -1,6 +1,7 @@
 resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
 resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 resolvers += Resolver.typesafeRepo("releases")
+resolvers += Resolver.jcenterRepo
```

Comment:
I'm a little nervous about including Jcenter - it's been sunset and while they're keeping it up as a read only mirror for now, it could disappear at somepoint: https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

What dependnecies required Jcenter?

Reply:
This was in build.sbt so I moved it here instead. I tried removing it and running the app which doesn't not appear to cause any issues so perhaps we should remove it?

Reply:
I see - if we're not using it and everything works (including tests) without, then let's drop it.